server/src/ScriptRunner/Scripts/index/add_not_investigated_to__investigation_statuses.sql

```diff
@@ -0,0 +1,27 @@
+INSERT INTO investigation_status (display_name, id)
+SELECT 'לא נחקר',100000005
+WHERE
+    NOT EXISTS (
+        SELECT id FROM investigation_status WHERE id = '100000005'
+    );
+
+INSERT INTO investigation_sub_status (display_name, id, parent_status)
+SELECT 'מדיניות משרד הבריאות',100000008,100000005
+WHERE
+    NOT EXISTS (
+        SELECT id FROM investigation_sub_status WHERE id = 100000008
+    );
+
+INSERT INTO investigation_sub_status (display_name, id, parent_status)
+SELECT 'מדיניות לשכה',100000009,100000005
+WHERE
+    NOT EXISTS (
+        SELECT id FROM investigation_sub_status WHERE id = 100000009
+    );
+
+INSERT INTO investigation_sub_status (display_name, id, parent_status)
```

Comment:
I'm not sure i understand whats your trying to do here

Reply:
@ornahasson read about ON CONFLICT:
https://www.postgresql.org/docs/9.5/sql-insert.html